File: Panagram Checking/Panagram Checking.py
class Solution:
    
    #Function to check if a string is Pangram or not.
    def checkPangram(self,s):
        a = list('abcdefghijklmnopqrstuvwxyz')
        s = s.lower()
        s = sorted(list(set(s)))
        if s[-26:] == a:
            return True
        else:
            return False

File: Panagram Checking/Panagram Checking.cpp
#include <iostream>
using namespace std;

class Solution
{
  public:
    //Function to check if a string is Pangram or not.
    bool checkPangram (string s) {
    int letters[26] = {0};
    for(char c: s)
        if(isalpha(c))
            letters[tolower(c) - 97]++;
    for(int i=0; i<26; i++)
        if(letters[i] < 1)
            return false;
    return true;        
    }

};